src/cmake/FindAPR-Util.cmake

```diff
@@ -34,7 +34,14 @@ find_program(APR_UTIL_CONFIG_EXECUTABLE
 mark_as_advanced(APR_UTIL_CONFIG_EXECUTABLE)
 if(EXISTS ${APR_UTIL_CONFIG_EXECUTABLE})
     _apu_invoke(APR_UTIL_INCLUDE_DIR   --includedir)
-    _apu_invoke(APR_UTIL_LIBRARIES     --link-ld)
+    if (APU_STATIC OR NOT BUILD_SHARED_LIBS)
+      _apr_invoke(_apr_util_link_args  --link-ld)
```

Comment:
Is `_apr_invoke` really correct here when everything else uses `_apu_invoke`?

Reply:
Opps -Sould all be 'apu' - I will fix those typos